k9mail/src/test/java/com/fsck/k9/view/ShowSenderTest.java

```diff
@@ -0,0 +1,56 @@
+package com.fsck.k9.view;
+
+import android.util.AttributeSet;
+
+import com.fsck.k9.K9RobolectricTestRunner;
+import com.fsck.k9.mail.Address;
+import com.fsck.k9.mail.Message;
+import com.fsck.k9.mail.internet.MimeMessage;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RuntimeEnvironment;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+import static org.mockito.Mockito.mock;
+
+@RunWith(K9RobolectricTestRunner.class)
+public class ShowSenderTest {
```

Comment:
Tests should be named `<ClassBeingTested>Test`. So this should be `MessageHeaderTest`